test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -175,6 +178,23 @@ export default class PerfCompare extends Component {
         })
     }
 
+    async getChildrenList ( resBenchmarkBuildJson ) {
```

Comment:
We can move the Jenkins specific code from `handleGenerateTable` and prepare the output here. Hence, this function would need a rename.

Reply:
updated!